src/components/app.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+
+import SearchForm from './search-form';
+
+import '../styles/app.css';
+
+
+const App = () =>   {
+  const [selectedDate, setSelectedDate] = useState(0);
+  
+  const [forecasts, setForecasts] = useState([]);
+
+  const [location, setLocation] = useState({
+    city: "",
+    country: "",
+  });
+
+  const [load, setLoad] = useState(false);
+
+  const selectedForecast = forecasts.find(forecast => forecast.date === selectedDate);
+
+  const handleForecastSelect = (date) => {
+    setSelectedDate(date);
+  }
+
+  
+
+  useEffect(() => {
+    axios
+    .get("https://mcr-codes-weather.herokuapp.com/forecast/")
+    .then((response) => {
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location); 
+      setLoad(true);  
```

Comment:
you might be using load differently to my assumption, but I would assume you wouldnt want to set to loading just before you do your axios call unless you are using it to state that the data has loaded. Either way I would use the doing verb to make it clearer for other people reading your code.